recipe/meta.yaml

```diff
@@ -19,7 +19,6 @@ outputs:
       noarch: python
       entry_points:
         - black = black:patched_main
-        - black-primer = black_primer.cli:main
```

Comment:
`black-primer` was dropped upstream in PR ( https://github.com/psf/black/pull/2924 )